src/Map.js

```diff
@@ -7,1005 +7,987 @@
+ * @property {String} property - layer property name that has changed
+ * @property {String} oldValue - old layer property value
+ * @property {String} newValue - new layer property value
+ */
+var layerChangedEvent = {
+};
+
+/**
+ * **Common 2D/3D object**
+ *
+ * Geolocated Event. Triggered when the map's center has changed by user geo-localization.<br/>
+ * One can listen to this event with [Gp.Map.listen()](Gp.Map.html#listen) method
+ *
+ * @namespace
+ * @alias Gp.GeolocatedEvent
+ *
+ * @property {Gp.Point} position - the position returned by geolocalisation
+ */
+var geolocatedEvent = {
+};
+
+/**
+ * **Common 2D/3D event**
+ *
+ * Located Event. Triggered when the map's center has changed by geocoding.<br/>
+ * One can listen to this event with [Gp.Map.listen()](Gp.Map.html#listen) method
+ *
+ * @namespace
+ * @alias Gp.LocatedEvent
+ *
+ * @property {Gp.Point} position - the position returned by localisation
+ */
+var locatedEvent = {
+};
+
+/**
+ * **Common 2D/3D event**
+ *
+ * Configured Event. Triggered when the map has finished to configure itself with the given apiKey at startup.<br/>
+ * One can listen to this event with [Gp.Map.load()](module-Map.html) function through the **mapEventsOptions** property of {@link Gp.MapOptions} parameter.<br/>
+ * *NB* : for a full map availability at startup you'd better listen to {@link Gp.MapLoadedEvent}.
+ *
+ * @namespace
+ * @alias Gp.ConfiguredEvent
+ *
+ * @property {Gp.Services.GetConfigResponse} config - Geoportal access configuration. See [Gp.Services.GetConfigResponse](https://ignf.github.io/geoportal-access-lib/latest/jsdoc/Gp.Services.GetConfigResponse.html) definition in geoportal-access-lib documentation.
+ */
+var configuredEvent = {
+
+};
+/**
+ * **Common 2D/3D event**
+ *
+ * Map Loaded Event. Triggered at startup when the map has finished to load and is ready to be commanded with {@link Gp.Map} object methods.<br/>
+ * One can listen to this event with [Gp.Map.load()](module-Map.html) function through the **mapEventsOptions** property of {@link Gp.MapOptions} parameter.<br/>
+ *
+ * @namespace
+ * @alias Gp.MapLoadedEvent
+ *
+ * @property {Gp.Map} map - loaded Map
+ */
+var mapLoadedEvent = {
+};
+
+/**
+ * Map options object.
+ *
+ * Holds options for map creation when using [Gp.Map.load()](module-Map.html) function.
+ *
+ * Availables options are described below.
+ *
+ * * ** Common 2D/3D properties**
+ *
+ * | property | Type | Argument | Default | Description |
+ * | - | - | - | - | - |
+ * | apiKey | String / Array.<String> | | | access key(s) to Geoportal platform, obtained [here](http://professionnels.ign.fr/ign/contrats) |
+ * | viewMode | String | optional | "2d" | Library to load the map with. Values are : <ul> <li> "2d" (default) to load the map with OpenLayers (2D) </li> <li>"3d" to be able to use 2D (OpenLayers) and 3D (iTowns) views. In this case, you have to use GpSDK3D.js file (instead of GpSDK2D.js)</li></ul> |
```

Comment:
Attention c'est un "breaking change" par rapport à la version précédente, où il fallait utiliser "library" ! => changement d'interface. 
Je suis plutôt d'accord avec ce changement, mais il faudra bien le préciser dans la release.

Reply:
C'est noté ;)